job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/QueueServices.java

```diff
@@ -106,15 +88,12 @@ public void sendMessage(
                 getStatusCheckIntervalMillis(ScheduledExecutorConfig.getStatusCheckIntervalSeconds()),
                 statusCheckUrl, ScheduledExecutorConfig.getTrackingPipe(), workerAction.getTargetPipe());
 
-        final QueueTaskMessage taskMessage = new QueueTaskMessage(
-                taskId,
-                workerAction.getTaskClassifier(),
-                workerAction.getTaskApiVersion(),
-                taskData,
-                TaskStatus.NEW_TASK,
-                Collections.<String, byte[]>emptyMap(),
-                targetQueue,
-                trackingInfo);
+        final String messageFormatVersion = ScheduledExecutorConfig.useNewQueueMessageFormat();
```

Comment:
Note that I moved that next to where it's used. We could even inline it but I thought it would make code easier to read.